.gitignore

```diff
@@ -1,9 +1,13 @@
 # Please keep this list in ASCII sort order
+.classpath
 .DS_Store
+.factorypath
 .idea
+.project
+.settings/
 src/main/webapp/.cache
 src/main/webapp/dist
 src/main/webapp/node
 src/main/webapp/node_modules
 step240-2020.iml
-target/
+target/
```

Comment:
nit: add blank line at the end of the file

Reply:
Rectified with ac0a3ac